app/assets/javascripts/home.coffee

```diff
@@ -26,19 +26,46 @@ set_coords_by_geolocation = (position) ->
   set_coords(position.coords.accuracy, position.coords.latitude, position.coords.longitude)
 
 block_callback = (err) ->
-  location_error()
+  if $('#location_geolocation').prop('checked')
+    $('#location_button').prop('innerHTML', 'Get My Location')
```

Comment:
Do we need to set this text dynamically, or could we just write "Get My Location" directly in the HTML? If we *do* need to set it dynamically, it would be better to just use jQuery's [`.html()`](https://api.jquery.com/html/) method.

Whatever adjustment is made here should also be made on line 60.

Reply:
Don't worry about this change for right now. What you have works just fine, so let's go with that!